docs/general/builtins/scalar-functions.rst

```diff
@@ -797,6 +797,25 @@ Returns: ``text``
     +--------+
     SELECT 1 row in set (... sec)
 
+.. _scalar-strpos:
+
+``strpos(string, substring)``
+-----------------------------
+
+Finds the first occurence index of substring in a string If substring does not exists it returns 0.
+
+Returns: ``integer``
+
+::
+
+    cr> SELECT strpos('crate' , 'ate');
+    +---+
+    | 3 |
+    +---+
+    | 3 |
+    +---+
+    SELECT 1 row in set (0.012 sec)
+
```

Comment:
```suggestion
    | 3 |
    +---+
    SELECT 1 row in set (... sec)

```

Reply:
We shouldn't use concrete numbers for execution time as it's deterministic and causes doc test failure.